src/Integration.Vsix/AsmRef_Integration.Vsix_Baseline_WithStrongNames.txt

```diff
@@ -122,12 +122,10 @@ Referenced assemblies:
 - 'System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'
 - 'System.ComponentModel.Composition, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'
 - 'System.Core, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'
-- 'System.Interactive.Async, Version=3.0.1000.0, Culture=neutral, PublicKeyToken=94bc3704cddfc263'
```

Comment:
Deleting the classes has changed the list of assemblies, but hasn't changed the project file or what ships in the VSIX.

* `System.IO.Copmression.FileSystem` wasn't directly referenced and wasn't included in the VSIX anyway, and
* `System.Interactive.Async` is a special case; a specific version is explicitly included in the VSIX because it's a dependency of some of the Google protobuf stuff. There's commented section in the project file explaining it. I'm not sure if we can drop it yet or not.